src/drawmembrane/draw_membrane2a.c

```diff
@@ -501,46 +529,64 @@ int main(int argc, char *argv[])
   float tmp_x,tmp_y,tmp_z,dx,dy,dz,l_c_x,l_c_y,l_c_z;
   float x0_p, y0_p, z0_p;
   float x0_x, y0_x, z0_x;
-  float x0_y, y0_y, z0_y; 
-  float x0_z, y0_z, z0_z; 
+  float x0_y, y0_y, z0_y;
+  float x0_z, y0_z, z0_z;
   float x0, y0, z0;
   float V=0, I=0.1, sdie, cdie, pdie, mdie, idie;
   float l_m=40, a0=0;
   float z_m0=0, R_m0=0, R_m1=0, x0_R=NO_COORDINATE, y0_R=NO_COORDINATE;
-  float R, R_temp;
+  float R, R2, R_temp;
   char *infix;
   char *file_name_x, *file_name_y, *file_name_z;
   char *file_name_k, *file_name_c;
   char *f1, *f2, *f3, *f4, *f5, *f6;
   char ext[]="m.dx";
   bool compression = FALSE;
+
+  /* Draw symmetric exclusion zone for a dimer. */
+  /* The zone is defined by x0_R, y0_R, dx_R, dy_R. */
+  bool dimer = FALSE; /* New feature for CPA dimer. Set to FALSE as default*/
```

Comment:
Change formatting and comment.
```suggestion
  bool dimer=FALSE; /* dimeric proteins with symmetric exclusion zones. Set to FALSE as default*/
```